css/style.css

```diff
@@ -0,0 +1,152 @@
+* {
+	box-sizing: border-box;
+	font-family: "Times New Roman", Times, serif;
+}
+
+h1 {
+	margin-bottom: 40px;
+	text-align: center;
+	font-weight: bold;
+	font-family: 'Alice', serif;
+}
+
+section {
+	position: relative;
+	font-weight: bold;
+	}
+
+#boxchicken {
```

Comment:
Please use snake case for selector names.  https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Reply:
Snake case is the incorrect term, correction has been done to add in hyphens.

Reply:
Apologies kebab-case :-D